### ISSUE TITLE
Symfony 6 upgrade failure

### ISSUE THREAD

**Issue by 7system7** [php-enqueue/enqueue-dev#1221]
If you would like to update to Symfony 6, there is an error:

## Environment

```console
❯ php --version
PHP 8.0.13 (cli) (built: Nov 22 2021 09:51:03) ( NTS )
Copyright (c) The PHP Group
Zend Engine v4.0.13, Copyright (c) Zend Technologies
    with Zend OPcache v8.0.13, Copyright (c), by Zend Technologies
❯ composer --version
Composer version 2.1.12 2021-11-09 16:02:04
❯ uname -a
Linux AMBER 5.13.0-21-generic #21-Ubuntu SMP Tue Oct 19 08:59:28 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

## The failure

Problem 1, and 2. :)

```composer
❯ composer upgrade
Loading composer repositories with package information
Restricting packages listed in "symfony/symfony" to "6.0.*"
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires enqueue/enqueue-bundle ^0.10.12 -> satisfiable by enqueue/enqueue-bundle[0.10.12].
    - enqueue/enqueue-bundle 0.10.12 requires symfony/framework-bundle ^4.3|^5 -> found symfony/framework-bundle[v4.3.0, ..., v4.4.34, v5.0.0, ..., v5.4.0] but it conflicts with your root composer.json require (6.0.*).
  Problem 2
    - Root composer.json requires enqueue/simple-client ^0.10.9 -> satisfiable by enqueue/simple-client[0.10.9].
    - enqueue/simple-client 0.10.9 requires symfony/config ^4.3|^5 -> found symfony/config[v4.3.0, ..., v4.4.34, v5.0.0, ..., v5.4.0] but these were not loaded, likely because it conflicts with another require.
  Problem 3
    - Root composer.json requires sroze/messenger-enqueue-transport ^0.5.1 -> satisfiable by sroze/messenger-enqueue-transport[0.5.1].
    - sroze/messenger-enqueue-transport 0.5.1 requires symfony/messenger ^4.3|^5 -> found symfony/messenger[v4.3.0, ..., v4.4.34, v5.0.0, ..., v5.4.0] but it conflicts with your root composer.json require (6.0.*).
  Problem 4
    - api-platform/core[v2.6.0, ..., v2.6.6] require symfony/http-foundation ^4.4 || ^5.1 -> found symfony/http-foundation[v4.4.0, ..., v4.4.34, v5.1.0, ..., v5.4.0] but these were not loaded, likely because it conflicts with another require.
    - Root composer.json requires api-platform/core ^2.6 -> satisfiable by api-platform/core[v2.6.0, ..., v2.6.6].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
```

**Comment by mamchyts** [comment]
+1

**Comment by mamchyts** [comment]
any updates?

**Comment by 7system7** [comment]
I started to use the native code (amqplib) for RPC connections and Symfony Messenger for anything else. Works like charm.

I had to figure out - again - that I do not need any 3rdparty bundle for a process what is working by itself. For most cases the Symfony's bundles are more than enough to solve the problems. They are simple & maintained quickly.

Good luck for the upgrade and happy new year! :))

**Comment by andrewmy** [comment]
Fixed by #1239
